tests/Provider/AttributeBagProvider.php

```
@@ -68,34 +120,208 @@ public static function invalidManyKey(): array
+            'without prefix ng' => [
+                'if',
+                'ng-',
+                'ng-if',
+            ],
+            'without prefix event' => [
+                'click',
+                'on',
+                'onclick',
+            ],
+        ];
+    }
```

PHPMD (2.15.0)

[warning] The method key() has 105 lines of code. Current threshold is set to 100. Avoid really long methods. (undefined)

(ExcessiveMethodLength)